### PR TITLE
docs(gatsby-plugin-manifest): Fixing gatsby-plugin-manifest README

### PR DESCRIPTION
**Pull request by derjp**
## Description

- Added missing punctuation, words, and fixed phrasing.
- Made hyperlinks more semantic (https://www.gatsbyjs.org/contributing/gatsby-style-guide/#use-clear-hyperlinks).
- Fixed abbreviations and terms that weren't spelled correctly (https://www.gatsbyjs.org/contributing/gatsby-style-guide/#abbreviate-terms).
- Replaced hyphens with em dashes wherever appropriate.
- Few, careful stylistic changes (text formatting).

I made all changes in accordance with the guidelines at https://www.gatsbyjs.org/contributing/gatsby-style-guide/ and https://www.thepunctuationguide.com/ and to the best of my knowledge. I also compared to how other plugins’ README files and the official Gatsby docs are written.

Since I'm not a native speaker though, there may be some errors, so I'd be glad if you point out any changes you don't agree with.

I plan on going over more documentation, but I would first like to get some feedback so I know whether I'm doing this correctly and whether these changes are wanted. :)

### CODE REVIEW

**Comment by laurieontech** [comment]
Hi @derjp, thank you so much for your time in submitting this PR and going through this doc so thoroughly. While there are some things I think it would be valuable to edit and merge, this is probably a bit more than it needs. I'm going to mark the changes that would be most valuable and you can feel free to edit this PR or open a new one with just those changes.

Thanks again for taking the time to contribute to Gatsby!

File: packages/gatsby-plugin-manifest/README.md [hunk 1]
```diff
@@ -1,17 +1,17 @@
 # gatsby-plugin-manifest
 
-The web app manifest(part of the [PWA](https://developer.mozilla.org/en-US/docs/Web/Progressive_web_apps) specification) enabled by this plugin allows users to add your site to their home screen on most mobile browsers —
+The web app manifest (part of the [PWA](https://developer.mozilla.org/en-US/docs/Web/Progressive_web_apps) specification) enabled by this plugin allows users to add your site to their home screen on most mobile browsers —
```
Comment (laurieontech) [inline packages/gatsby-plugin-manifest/README.md]: Let's keep this!

File: packages/gatsby-plugin-manifest/README.md [hunk 2]
```diff
@@ -48,40 +48,40 @@ the created `manifest.webmanifest`.
...
+  - Favicon — _never_
+  - Legacy icon support — _yes_
+  - Cache busting — _never_
+  - Localization — _optional_
 
 **_IMPORTANT:_** For best results, if you're providing an icon for generation it should be...
 
 - ...at least as big as the largest icon being generated (512x512 by default).
-- ...square (if it's not, transparent bars will add to make it square).
+- ...square (if it's not, transparent bars will automatically be added to make it square).
```
Comment (laurieontech) [inline packages/gatsby-plugin-manifest/README.md]: Let's keep this

File: packages/gatsby-plugin-manifest/README.md [hunk 3]
```diff
@@ -109,13 +109,13 @@ Add the following line to the plugin options
   ], // Add or remove icon sizes as desired
 ```
 
-If you want to include more or fewer sizes, then the hybrid option is for you. Like automatic mode, you include a high-resolution icon from which to generate smaller icons. But unlike automatic mode, you provide the `icons` array config and icons are generated based on the sizes defined in your config. Here's an example `gatsby-config.js`:
+If you want to include more or fewer sizes, then the hybrid option is for you. Like automatic mode, you include a high-resolution icon from which to generate smaller icons. But unlike automatic mode, you provide the `icons` array config and icons are generated based on the sizes defined in your config.
```
Comment (laurieontech) [inline packages/gatsby-plugin-manifest/README.md]: Let's keep this

File: packages/gatsby-plugin-manifest/README.md [hunk 4]
```diff
@@ -134,17 +134,17 @@ icons: [
 
 In the manual mode, you are responsible for defining the entire web app manifest and providing the defined icons in the [static](https://www.gatsbyjs.org/docs/static-folder/) folder. Only icons you provide will be available. There is no automatic resizing done for you.
 
-### Feature configuration - **Optional**
+### Feature configuration - _optional_
 
 #### Localization configuration
 
-Localization allows you to create unique manifests for each localized version of your site. As many languages as you want are supported. Localization requires unique paths for each language (e.g. if your default about page is at `/about`, the German (`de`) version would be `/de/about`)
+Localization allows you to create unique manifests for each localized version of your site. You can add as many languages as you want. Localization requires schematic paths for each language (e.g. if your default about page is at `/about`, the German (`de`) version would be `/de/about`).
```
Comment (laurieontech) [inline packages/gatsby-plugin-manifest/README.md]:
```suggestion
Localization allows you to create unique manifests for each localized version of your site. You can add as many languages as you want. Localization requires unique paths for each language (e.g. if your default about page is at `/about`, the German (`de`) version would be `/de/about`).
```
I'd recommend keeping the first change but reverting the second.

File: packages/gatsby-plugin-manifest/README.md [hunk 5]
```diff
@@ -134,17 +134,17 @@ icons: [
...
+`lang` and `start_url` are the only _required_ options in the array objects. `name`, `short_name`, and `description` are [recommended](https://www.w3.org/TR/appmanifest/#dfn-directionality-capable-members) to be translated if being used in the default language configuration. All other config options are optional. This is helpful if you want to provide unique icons for each locale.
 
-The [`lang` option](https://www.w3.org/TR/appmanifest/#lang-member) is part of the web app manifest specification and thus is required to be a [valid language tag](https://www.iana.org/assignments/language-subtag-registry/language-subtag-registry)
+The [`lang` option](https://www.w3.org/TR/appmanifest/#lang-member) is part of the web app manifest specification and thus is required to be a [valid language tag](https://www.iana.org/assignments/language-subtag-registry/language-subtag-registry).
```
Comment (laurieontech) [inline packages/gatsby-plugin-manifest/README.md]: Let's keep this

File: packages/gatsby-plugin-manifest/README.md [hunk 6]
```diff
@@ -213,7 +213,7 @@ module.exports = {
 
 #### Disable legacy icons
 
-iOS 11.3 added support for the web app manifest spec. Previous iOS versions won't recognize the icons defined in the webmanifest and the creation of `apple-touch-icon` links in `<head>` is needed. This plugin creates them by default. If you don't want those icons to be generated you can set the `legacy` option to `false` in plugin configuration:
+iOS 11.3 added support for the web app manifest specification. Previous iOS versions won't recognize the icons defined in the webmanifest and the creation of `apple-touch-icon` links in `<head>` is needed. This plugin creates them by default. If you don't want those icons to be generated you can set the `legacy` option to `false` in the plugin configuration:
```
Comment (laurieontech) [inline packages/gatsby-plugin-manifest/README.md]: Let's keep this

File: packages/gatsby-plugin-manifest/README.md [hunk 7]
```diff
@@ -269,7 +269,7 @@ Cache busting works by calculating a unique "digest" of the provided icon and mo
 
 **Options:**
 
-- **\`query\`** - This is the default mode. File names are unmodified but a URL query is appended to all links. e.g. `icons/icon-48x48.png?digest=abc123`
+- **\`query\`** - This is the default mode. File names are unmodified but a URL query is appended to all links. e.g. `icons/icon-48x48.png?digest=abc123`.
```
Comment (laurieontech) [inline packages/gatsby-plugin-manifest/README.md]: Let's keep this

File: packages/gatsby-plugin-manifest/README.md [hunk 8]
```diff
@@ -412,9 +412,9 @@ Internet Explorer is the only other major browser that doesn't support the web a
 ### Additional resources
 
 This article from the Chrome DevRel team is a good intro to the web app
-manifest—https://developers.google.com/web/fundamentals/engage-and-retain/web-app-manifest/
+manifest — https://developers.google.com/web/fundamentals/engage-and-retain/web-app-manifest/.
```
Comment (laurieontech) [inline packages/gatsby-plugin-manifest/README.md]: Let's keep this

File: packages/gatsby-plugin-manifest/README.md [hunk 9]
```diff
@@ -412,9 +412,9 @@ Internet Explorer is the only other major browser that doesn't support the web a
...
 
-For more information see the w3 spec https://www.w3.org/TR/appmanifest/ or Mozilla docs https://developer.mozilla.org/en-US/docs/Web/Manifest.
+For more information see the [W3C specification](https://www.w3.org/TR/appmanifest/) or [Mozilla documentation](https://developer.mozilla.org/en-US/docs/Web/Manifest).
```
Comment (laurieontech) [inline packages/gatsby-plugin-manifest/README.md]: And keep this

**Comment by derjp** [comment]
Thank you for going over my suggestions and for your thorough feedback. :) I overrode my last commit with a new one that only includes the necessary changes (I hope nothing broke due to the forced push).

**Comment by laurieontech** [comment]
This is perfect! Thank you so much.

**Pull request merged**